extra_data/validation.py

```diff
@@ -177,39 +204,38 @@ def run_checks(self):
         self.check_files_map()
         return self.problems
 
-    def progress(self, line):
+    def progress(self, done, total, fname, nproblems):
         """Show a line of progress information"""
         if not self.term_progress:
             return
 
+        line = progress_bar(done, total)
+        line += f'\n{nproblems} problems: {fname}'
```

Comment:
Thanks for responding to that comment, but this won't do quite what I meant. Displaying the *next* file it's about to check is useful information if it gets stuck. I don't think it's worth showing the last checked file - it doesn't mean much, and it can be misleading if people think that's currently being checked.

I think this is just something we lose by parallelising the code. The massive speedup is worth a minor trade-off!

Reply:
What about something like:

```
Checking run directory: /gpfs/exfel/exp/XMPL/201750/p700000/raw/r0006
Progress: 21/21 [##################################################]
128 problems in 16 files (last: RAW-R0803-AGIPD03-S00000.h5)
```

?

Reply:
I'm inclined to drop the filename altogether - I'm not sure the last file checked tells you anything useful.

Reply:
ah, it's not the last checked file, it's the last checked file that has problems

Reply:
OK, fair enough.

Reply:
I mean, if you think that it's not useful, I'll remove it, problems are showed at the end anyway (or reword if that's not clear enough that's showing file with problems)

Reply:
I'm not sure. Last with problems seems more likely to be useful than last checked, but I've been thinking about it too long to know if the meaning is clear for other people. I'm happy for this to go in either with or without that.

Reply:
Alright then, it stays for now :)